etc/config/crystal.amazon.properties

```diff
@@ -1,7 +1,7 @@
 compilers=&crystal
 defaultCompiler=crystal131
```

Comment:
@HertzDevil can you fix the default compiler please

```
Found 1 property file(s) with mismatching ids:
./etc/config/crystal.amazon.properties
UNKNOWN DEFAULT COMPILER:
	crystal131
```

Reply:
+1 for the script catching this, good thing it's there!